test/typings/either-spec.ts

```diff
@@ -57,7 +57,7 @@ log(unpacked).run();
 log2(getMessage(null).flatMap(getType)).performUnsafeIO();
 log3(getMessage({type: 'x', payload: null}).chain(getType));
 
-const nameError: string = Left('-- Adamovisch').leftMap(n => n.split(' ').shift()).left();
+const nameError: string = Left('-- Adamovisch').leftMap((n:string) => n.split(' ').shift() || "").left();
```

Comment:
TS should infer properly type of `n`, does it work without type annotation for `n` ?